my-app/src/components/SearchBar.css

```diff
@@ -25,13 +34,43 @@
 }
 
 #submitButton {
-    border: none;
-    border-radius: 25px;
-    background-color: #75E6DA;
-    vertical-align: middle;
-    margin-left: 0.5%;
-    height: 3em;
-    width: 18%;
-    font-size: 20px;
-    font-weight: bold;
-}
\ No newline at end of file
+  border: none;
+  border-radius: 25px;
+  background-color: #75E6DA;
+  vertical-align: middle;
+  margin-left: 0.5%;
+  height: 3em;
+  width: 15%;
+  font-size: 20px;
+  font-weight: bold;
+}
+
+#navbar {
```

Comment:
This might look better centered thoughts?

Reply:
I agree, this was just boilerplate from some tutorial lol, will make an edit at some point to centre it before i merge